38-audio-classifier/README.md

```diff
@@ -0,0 +1,220 @@
+# Audio Classifier
+
+Monitor the environment for a particular sound and publish the detections to Notehub.
+
+- [Audio Classifier](#audio-classifier)
+  - [Overview](#overview)
+  - [You Will Need](#you-will-need)
+    - [Hardware](#hardware)
+    - [Software](#software)
+  - [Hardware Setup](#hardware-setup)
+  - [Building the Audio Classifier Model](#building-the-audio-classifier-model)
+    - [Collecting Data](#collecting-data)
+      - [Prebuilt Dataset](#prebuilt-dataset)
+      - [Custom Dataset](#custom-dataset)
+    - [Training the Model](#training-the-model)
+    - [Downloading the Model](#downloading-the-model)
+  - [Notehub Setup](#notehub-setup)
+    - [Environment Variables](#environment-variables)
+    - [`detections.qo`](#detections-qo)
+  - [Classifier Firmware](#firmware-setup)
+  - [Testing](#testing)
+  - [Blues Community](#blues-community)
+
+## Overview
```

Comment:
Trivial, but I’d break this overview’s contents into two paragraphs for readability.